### DISCUSSION TITLE
can't  install   zigbee2mqtt on  home  assistant  os

### DISCUSSION THREAD

**Discussion by patrick-L91** [Koenkk/zigbee2mqtt#14966] [Devices]
i have installed  home  assistant os  and  zigbee2mqtt 
```
Home Assistant 2022.11.2
Supervisor 2022.10.2
Operating System 9.3
Interface utilisateur : 20221108.0 - latest
```

```
Zigbee2MQTT
Current version: 1.28.2-1 (Journal des modifications)
```
I put the configuration of the zigbee2mqtt.io site but it apparently concerns the docker version of home assistant
configuration yaml in zigbee2mqtt addon
```
data_path: /config/zigbee2mqtt
socat:
  enabled: false
  master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
  slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
  options: "-d -d"
  log: false
  permit_join: true
  frontend: true
mqtt:
  mqtt:
    server: mqtt://192.168.10.11:1883
serial:
  port: /dev/ttyUSB0
zigbee_herdsman_debug: false
```
and  the  error messages : 
```
Zigbee2MQTT
[23:15:32] INFO: Preparing to start...
[23:15:32] INFO: Socat not enabled
[23:15:34] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:error 2022-11-13 23:15:50: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-11-13 23:15:50: Failed to start zigbee
Zigbee2MQTT:error 2022-11-13 23:15:50: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-11-13 23:15:50: Exiting...
Zigbee2MQTT:error 2022-11-13 23:15:50: Error: AREQ - ZDO - activeEpRsp after 10000ms
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
```

**Comment by piasekps** [comment]
I just got exactly the same error. It was working before. What I did was install ZHA next to zigbee2mqtt with the second coordinator. I start pairing devices and then I remove ZHA. And from that moment zigbee doesn't wake up anymore raising the exact same error as above.

**Reply by piasekps** [reply]
It starts working again I just remove the new coordinator, and the old one unplugs and plugin again.

**Reply by patrick-L91** [reply]
tank's  for you

I have done   a  hardware   reset   and  i have  the  frontend....   
I think the problems are between the configuration of the addon... and what we put in / config / gzigbee2mqtt / configuration.yaml.

but  actually  i can't  not   paring   the  bulb...  how to do  to  force  the   bulb  to  paring.
